flytekit/core/python_auto_container.py

```diff
@@ -253,4 +253,4 @@ def get_registerable_container_image(img: Optional[str], cfg: ImageConfig) -> st
 # fqn will access the fully qualified name of the image (e.g. registry/imagename:version -> registry/imagename)
 # version will access the version part of the image (e.g. registry/imagename:version -> version)
 # With empty attribute, it'll access the full image path (e.g. registry/imagename:version -> registry/imagename:version)
-_IMAGE_REPLACE_REGEX = re.compile(r"({{\s*\.image[s]?(?:\.([a-zA-Z]+))(?:\.([a-zA-Z]+))?\s*}})", re.IGNORECASE)
```

Comment:
the [docker tag docs](https://docs.docker.com/engine/reference/commandline/tag/#description) don't offer a regex. That said, I'm ok with this simpler regex, even if it doesn't adhere to the spec completely.